### DISCUSSION TITLE
Transferring Labels/Poses onto Newly Encoded Video

### DISCUSSION THREAD

**Discussion by jmdelahanty** [talmolab/sleap#926] [Help!] [answered]
Hello sleapyheads!

I was curious if there's a way to easily reassign labels from one video to another. We had some trouble opening some HEVC encoded videos on our Linux distros with SLEAP so we re-encoded stuff to H264 with your recommended settings.

Is there a way to assign labels from one video to another/can we do that in this situation? Or is changing the encoding enough to make the labels from the previous video unusable on the new ones?

**Comment by talmo** [comment, marked as answer]
If the filenames are the same, you don't need to do anything. If not, you can go to File -> Replace videos and pick the new files. As long as the frame count is the same, there should be no issues.

**Reply by jmdelahanty** [reply]
Can report back if things worked out okay! If you take a frame encoded by HEVC and compare it to one encoded in H264 would they be the same? Because if they're different wouldn't the points it uses be different? Or at least appear different? I guess I can test this on my own, can do that sometime this week as an example...

**Reply by talmo** [reply]
Did you get a chance to test this?

In any case, the frame count and image size shouldn't change when you reencode, so this should hopefully not be an issue as everything (frame indices and image coordinates) will still line up with the previous video.

**Reply by jmdelahanty** [reply]
I'll double check with Laurel tomorrow and see if she's had a chance to use it on the converted videos.

**Reply by jmdelahanty** [reply]
It worked!
